### PR TITLE
feat: add contributors list on homepage [#72]

### PR DESCRIPTION
**Pull request by meshkat-hossain**
Resolved: #72 

- Add contributor list of Team component
- Implement a list of excluded users, where the excluded users removed in the section
image

### CODE REVIEW

**Comment by alaminsahed** [comment]
@meshkat-hossain Please resolve the merge conflicts in your PR. Take a pull request from the main branch before creating your PR.

OR

 You can follow these steps to resolve the merge conflicts issue. Go to your terminal and follow the commands.
```
1. git checkout main
2.  git pull
3. git checkout <your branch>
4. git merge main
5. git add . 
6. git commit -m "your message"
7. git push
```
Some additional resources may help you
https://www.youtube.com/watch?v=QmKdodJU-js
https://www.youtube.com/watch?v=lz5OuKzvadQ
https://www.youtube.com/watch?v=kBIMGOxqqnk

**Comment by meshkat-hossain** [comment]
Merge conflicts solved,, please check.

**Review by alaminsahed — APPROVED** [review]

**Comment by alaminsahed** [comment]
@mimukit This PR is good to go from my side. Please have a look.

File: src/components/home/Team.tsx [hunk 1]
```diff
@@ -117,6 +129,51 @@ export const Team = () => {
           </ul>
         </div>
       </div>
+      <div className="bg-white dark:bg-slate-900">
+        <div className="mx-auto max-w-7xl py-12 px-4 text-center sm:px-6 lg:px-8 lg:py-24">
+          <div className="space-y-8 sm:space-y-12">
+            <div className="space-y-5 sm:mx-auto sm:max-w-xl sm:space-y-4 lg:max-w-5xl">
+              <p className="text-2xl font-bold tracking-tight text-white ">
```
[the remark this answers is not in the document]
Comment (meshkat-hossain) [reply src/components/home/Team.tsx:136]: This size was suggested by mukit bro

**Comment by meshkat-hossain** [comment]
@alaminsahed vhai text-2xl recommended by mukit bro

**Pull request merged**